common/api/ecschema-locaters.api.md

```diff
@@ -19,38 +20,56 @@ export class FileSchemaKey extends SchemaKey {
     schemaText?: string;
 }
 
-// @beta
+// @alpha
 export abstract class SchemaFileLocater {
     constructor();
     addSchemaSearchPath(schemaPath: string): void;
     addSchemaSearchPaths(schemaPaths: string[]): void;
+    // (undocumented)
+    addSchemaText(schemaPath: string, readSchemaText: DelayedPromise<string | undefined>): Promise<void>;
+    // (undocumented)
+    addSchemaTextSync(schemaPath: string, readSchemaText: DelayedPromise<string | undefined>): void;
```

Comment:
it seems wrong that a sync method takes a delayed promise

Reply:
I removed the sync method since it was unnecessary, as we simply take a promise from the parameter and append it to an array.